diagnostics-app/angular.json

```diff
@@ -95,6 +96,7 @@
           "builder": "@angular-devkit/build-angular:karma",
           "options": {
             "main": "src/test.ts",
+            "codeCoverage": true,
```

Comment:
Is this related to the PR goal?

Reply:
My bad. This should've been in the Fix CI. I'll remove this from here and raise a new PR for fixing the hook variable name and adding this variable.

Reply:
Removed this from here. 👍🏻